evansmusomi/burger-builder
# Adds Order Button

*What this does*
- Adds "Order Now" button below build controls
- Ensures check out is disabled when no ingredients are added

*How to verify*
- User should be able to see the ORDER NOW button on the main page
- User should see a disabled (grey) button when they remove all ingredients
